tests/www/job_seekers_views/test_create_or_update.py

```diff
@@ -11,12 +12,109 @@
 from itou.utils.urls import add_url_params
 from itou.www.job_seekers_views.enums import JobSeekerSessionKinds
 from tests.companies.factories import CompanyFactory
+from tests.institutions.factories import InstitutionWithMembershipFactory
 from tests.prescribers.factories import PrescriberOrganizationWithMembershipFactory
-from tests.users.factories import JobSeekerFactory
+from tests.users.factories import ItouStaffFactory, JobSeekerFactory
 from tests.utils.test import KNOWN_SESSION_KEYS
 
 
-class TestCreateForJobSeeker:
+class TestGetOrCreateAsOther:
+    TUNNELS = ["sender", "hire", "gps"]
+
+    def test_labor_inspectors_are_not_allowed_to_get_or_create_job_seeker(self, client):
+        company = CompanyFactory()
+        institution = InstitutionWithMembershipFactory()
```

Comment:
Tu peux directement faire:
```suggestion
        institution_member = InstitutionMembershipFactory().user
```
pour éviter une requête SQL en faisant le `institution.members.first()`